app/src/components/base/Table/Table.tsx

```diff
@@ -60,47 +60,55 @@ export function Table<T extends unknown>({
       <Header>{header}</Header>
       <StyledTable>
         <TableHead>
-          {getHeaderGroups().map(headerGroup => (
-            <TableHeader key={headerGroup.id}>
-              {headerGroup.headers.map(header => (
-                <Th
-                  key={header.id}
-                  colSpan={header.colSpan}
-                  style={{ width: header.getSize() }}
-                  sortable={header.column.getCanSort()}
-                  onClick={() =>
-                    header.column.getCanSort()
-                      ? header.column.toggleSorting(
-                          header.column.getIsSorted() === 'asc',
-                        )
-                      : undefined
-                  }>
-                  {header.isPlaceholder ? null : (
-                    <>
-                      {flexRender(
-                        header.column.columnDef.header,
-                        header.getContext(),
+          {getHeaderGroups().map(headerGroup => {
+            return (
+              <TableHeader key={headerGroup.id}>
+                {headerGroup.headers.map(header => {
+                  const isSorted = header.column.getIsSorted();
+                  const canSort = header.column.getCanSort();
+
+                  return (
+                    <Th
+                      key={header.id}
+                      colSpan={header.colSpan}
+                      style={{ width: header.getSize() }}
+                      sortable={header.column.getCanSort()}
+                      onClick={() =>
+                        header.column.getCanSort()
+                          ? header.column.toggleSorting(
+                              header.column.getIsSorted() === 'asc',
+                            )
+                          : undefined
```

Comment:
NIT: Thoughts on adding a helper function on `.column` to handle this?

Reply:
Ah, probably in the future. This whole table file is a bit of mess, and eventually I'll go and clean it up - it's all the tanstack stuff, which is not super readable. If you're OK, I'll just keep as in for now?

Reply:
Yep works for me, you've got an approval :)